src/execution/sql/storage_interface.cpp

```diff
@@ -47,6 +48,23 @@ storage::ProjectedRow *StorageInterface::GetIndexPR(catalog::index_oid_t index_o
   return index_pr_;
 }
 
+bool StorageInterface::VerifyTableInsertConstraint() {
+  auto *pr = table_redo_->Delta();
+  return db_accessor_->VerifyTableInsertConstraint(table_oid_, pr);
+}
+
+bool StorageInterface::UpdateVerify() {
```

Comment:
Currently UpdataVerify is just returning true all the time. Need logic to actually verify.